numpy/lib/arraypad.py

```diff
@@ -1075,9 +1075,8 @@ def pad(array, pad_width, mode='constant', **kwargs):
     think about with a rank 2 array where the corners of the padded array
     are calculated by using padded values from the first axis.
 
-    The padding function, if used, should return a rank 1 array equal in
-    length to the vector argument with padded values replaced. It has the
-    following signature::
+    The padding function, if used, should modify a rank 1 array in-place. It
+    has the following signature::
```

Comment:
Might I suggest another quick fix to the docstring here? In the explanation of `padding_func`'s arguments the variable `pad_tuple` is used which doesn't appear in the signature. I guess it should be called `iaxis_pad_width` instead? (GitHub wouldn't let me comment in the actual place, line 1087)

Reply:
good catch. I have made the suggested change